content/blog/advanced-typescript-type-ftw/index.md

```diff
@@ -2,6 +2,7 @@
 title: "Advanced TypeScript type FTW!"
 authors: ["cyrus-najmabadi"]
 tags: ["TypeScript"]
+meta_desc: "TypeScript is great for cloud apps and infrastructure because of its rich type system. In this blog, Cyrus shows how the typesystem works for infrastructure as code in Pulumi. "
```

Comment:
```suggestion
meta_desc: "TypeScript is great for cloud apps and infrastructure because of its rich type system. In this blog, Cyrus shows how the type system works for infrastructure as code in Pulumi. "
```